src/test/java/net/openhft/chronicle/bytes/BytesInternalTest.java

```diff
@@ -327,6 +326,24 @@ public void testParseDouble() {
         }
     }
 
+    @Test
+    public void testCopyAfterSkip() {
+        final Bytes<byte[]> src = Bytes.from("hello again");
+
+        src.readSkip(7);
```

Comment:
I don't understand. Doesn't readSkip increase the read position? Shouldn't `src.copy()` be "gain"?

Reply:
I think so! But it used to be "hell" before fix.